src/experiments/doorhanger/api.js

```diff
@@ -75,9 +76,9 @@ class DoorhangerEventEmitter extends EventEmitter {
       learnMoreURL,
       escAction: "buttoncommand",
       eventCallback: doorhangerEvents,
-      removeOnDismissal: true,
+      removeOnDismissal: false,
     };
-    recentWindow.PopupNotifications.show(browser, "doh-first-time", text, null, primaryAction, secondaryActions, options);
+    let notification = recentWindow.PopupNotifications.show(browser, "doh-first-time", text, null, primaryAction, secondaryActions, options);
```

Comment:
No assignment here needed right?

Reply:
This is assigned to close it on the dismissed event _(Line 60 above in the same file)._

`recentWindow.PopupNotifications.remove(notification);`

Reply:
Won't the scope cause an issue here? Is it better to declare it before `let doorhangerEvents`?

Reply:
Revised!